src/tools/schema2base.ts

```diff
@@ -29,18 +30,19 @@ export abstract class Schema2Base {
   }
 
   private async processFile(src: string) {
+    Utils.init(`${process.cwd()}/${src}`);
```

Comment:
The first argument to `Utils.init` is supposed to be the relative path from CWD to arcs-root (or an absolute path to arcs-root).  Neither `${process.cwd()}` (and especially not `${process.cwd()}/${src}` is a 'path from CWD to arcs-root'.

This isn't causing it to fail because you haven't encountered a recipe with `$` macros inside. However, we probably don't want to leave this as a land-mine.